apps/frontend-pwa/package.json

```diff
@@ -17,18 +17,31 @@
     "@fortawesome/free-solid-svg-icons": "6.1.2",
     "@fortawesome/react-fontawesome": "0.2.0",
     "@klicker-uzh/graphql": "^2.0.0-alpha.8",
-    "@uzh-bf/design-system": "0.0.26",
+    "@radix-ui/react-progress": "^1.0.0",
+    "@socialgouv/matomo-next": "1.4.0",
+    "@uzh-bf/design-system": "0.0.30",
+    "dayjs": "1.11.5",
     "deepmerge": "4.2.2",
     "formik": "2.2.9",
     "graphql": "16.6.0",
+    "localforage": "^1.10.0",
     "next": "12.2.5",
     "next-pwa": "5.6.0",
     "ramda": "0.28.0",
     "react": "18.2.0",
+    "react-countdown-circle-timer": "3.0.9",
```

Comment:
might also move this to the design system

Reply:
removed unused dependency